.github/workflows/src/summarize-checks/summarize-checks.js

```diff
@@ -292,12 +293,13 @@ export default async function summarizeChecks({ github, context, core }) {
     return;
   }
 
+  // TODO: This is triggered by pull_request_target AND workflow_run.  If workflow_run, targetBranch will be undefined.
```

Comment:
Yes this is ok. `impactAssessment` publishes some additional relevant metadata about the triggering PR. `targetbranch` and `isDraft` are the two relevant to this comment.

Reply:
Frankly we should probably just remove the targetbranch log. Useless when this thing is actually doing work. 😂